src/crepe/api/LoopTimer.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include <cstdint>
+
+namespace crepe{
+class LoopTimer {
+public:
+/**
+ * \brief Get the singleton instance of LoopTimer.
+ *
+ * \return A reference to the LoopTimer instance.
+ */
+static LoopTimer& get_instance();
```

Comment:
Willen we maar 1 timer in onze game engine? Dus moet dit singleton zijn

Reply:
ik hoorde dat bob de delta time op ieder moment kon opvragen net zoals het in unity gebeurd ik dacht dat een singleton hiet het makkelijkste voor is